src/backend/aspen/workflows/nextstrain_run/nextstrain_profile/description_non_contextualized.md

```diff
@@ -1,2 +1,17 @@
 ### About this tree
 This tree is designed to provide an overview of samples collected only from within the user’s jurisdiction. This tree lacks contextual data that are crucial for making sound epidemiologic inferences, and thus we caution against its use for any purpose other than simply looking at viral diversity across the jurisdiction. These trees show all of the samples collected within a user’s jurisdiction by including data from CZ Gen Epi and GISAID, and if the total number of samples exceeds 2000, subsampling will occur and not all samples will be on the tree. 
+
+Site numbering and genome structure uses [Wuhan-Hu-1/2019](https://www.ncbi.nlm.nih.gov/nuccore/MN908947) as reference. The phylogeny is rooted relative to early samples from Wuhan. Temporal resolution assumes a nucleotide substitution rate of 8 &times; 10^-4 subs per site per year. Mutational fitness is calculated using results from [Obermeyer et al (under review)](https://www.medrxiv.org/content/10.1101/2021.09.07.21263228v1). Full details on bioinformatic processing can be found [here](https://github.com/nextstrain/ncov).
+
+
+### Acknowledgement
+We are grateful to the data contributors who shared the data used in this Web Application via the [GISAID Initiative](https://gisaid.org) (see citation): the Authors, the Originating Laboratories responsible for obtaining the specimens, and the Submitting Laboratories that generated the genetic sequences and metadata. An attribution table is available by clicking on "Download Data" at the bottom of the page and then clicking on "Acknowledgments" in the resulting dialog box.
+
+At the specific request of GISAID, we:
+ - maintain the prefix `hCoV-19/` in the names of viral isolates
+ - disable download of full metadata TSV and provide instead an acknowledgments TSV in the "Download Data" link at the bottom of the page
+ - refrain from sharing alignments or other intermediate files computed in our pipeline
+
+Data used in this web application remain subject to [GISAID’s Terms and Conditions](http://www.gisaid.org/DAA/).
+
+Citation: (a) Elbe, S., and Buckland-Merrett, G. (2017) Data, disease and diplomacy: GISAID’s innovative contribution to global health. Global Challenges, 1:33-46. DOI: 10.1002/gch2.1018 and / or (b) Shu, Y., McCauley, J. (2017) GISAID: From vision to reality. EuroSurveillance, 22(13) DOI: 10.2807/1560-7917.ES.2017.22.13.30494.
```

Comment:
since 3 files for 3 tree types all have the same info, in theory i could do it more programatic, but given the likelihood we'll never edit these files ever again, this should do for now.